ccd-definition/ComplexTypes/CareSupervision/RespondentNew-nonprod.json

```diff
@@ -23,5 +23,14 @@
     "ElementLabel": "Representative",
     "SecurityClassification": "Public",
     "FieldShowCondition": "legalRepresentation=\"Yes\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentNew",
```

Comment:
we don't need the policy reference anymore, since we are never deleting a respondent after submission